definitions/variable/industry/tag_iron_commodities.yaml

```diff
@@ -0,0 +1,15 @@
+- Iron Commodity:
+    - Steel:
+        description: raw steel, an alloy of carbon and iron
+
+- Iron Intermediate:
```

Comment:
There can only be one tag-list per yaml file.

Reply:
Sorry, I forgot about this. I assumed this is allowed since nomenclature does apparently parse all tag-lists if I instantiate a `DataStructureDefinition` from this branch.